lib/rules/base.js

```diff
@@ -65,6 +68,26 @@ module.exports = class Base {
     this.source = options.rawSource.match(reLines);
   }
 
+  get filePath() {
+    return this._filePath;
+  }
+
+  get resolvedFilePath() {
+    return path.resolve(this.workingDir, this.filePath);
+  }
+
+  get moduleName() {
+    return this._moduleName;
+  }
+
+  get rawSource() {
+    return this._rawSource;
+  }
+
+  get workingDir() {
+    return this._workingDir;
+  }
```

Comment:
I'd rather use `this.workingDir = options.workingDir` than make a specific getter for it, unless you have a particular reason for wanting it to be a getter...

Reply:
done